app/model/hf_runtime.py

```
@@ -0,0 +1,253 @@
+                text,
+                return_offsets_mapping=True,
+                truncation=True,
+                max_length=self._max_seq_len,
+                return_tensors="pt",
+            )
+            return enc
+
+        try:
+            enc = await asyncio.to_thread(_prepare_inputs)
+        except Exception as exc:
+            raise InferenceError("Tokenization failed") from exc
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)